modules/plottinggl/processors/scatterplotmatrixprocessor.cpp

```diff
@@ -68,26 +69,27 @@ ScatterPlotMatrixProcessor::ScatterPlotMatrixProcessor()
     , fontFaceStats_("fontFaceStats", "Font Face (stats)")
     , statsFontSize_("statsFontSize", "Font size (stats)", 14, 0, 144, 1)
     , showCorrelationValues_("showStatistics", "Show correlation values", true)
+    , parameters_("parameters", "Parameters")
     , correlectionTF_("correlectionTF", "Correlation TF")
-
+    
     , textRenderer_()
     , textureQuadRenderer_()
 
-    , mouseEvent_(
-          "mouseEvent", "Mouse Event",
-          [&](Event *e) {
-              if (auto me = dynamic_cast<MouseEvent *>(e)) {
-                  auto p = ivec2(me->posNormalized() * dvec2(static_cast<double>(numParams_))) + 1;
-                  if (p.x == p.y) {
-                      color_.setSelectedValue(p.x);
-                  } else {
-                      selectedX_.setSelectedValue(p.x);
-                      selectedY_.setSelectedValue(p.y);
-                  }
-              }
-
-          },
-          MouseButton::Left, MouseState::Press)
+    , mouseEvent_("mouseEvent", "Mouse Event",
+                  [&](Event *e) {
+                      if (auto me = dynamic_cast<MouseEvent *>(e)) {
+                          auto p =
+                              ivec2(me->posNormalized() * dvec2(static_cast<double>(numParams_)));
+                          if (p.x == p.y) {
+                              color_.setSelectedValue(visibleIDToColumnID_[p.x]);
+                          } else {
+                              selectedX_.setSelectedValue(visibleIDToColumnID_[p.x]);
+                              selectedY_.setSelectedValue(visibleIDToColumnID_[p.y]);
+                          }
+                      }
+
+                  },
+                  MouseButton::Left, MouseState::Press)
```

Comment:
why press and not release?

Reply:
It was like that before, did not think of chaning it.